cl/search/tasks.py

```diff
@@ -1376,3 +1409,75 @@ def index_related_cites_fields(
     if settings.ELASTICSEARCH_DSL_AUTO_REFRESH:
         # Set auto-refresh, used for testing.
         OpinionClusterDocument._index.refresh()
+
+
+@app.task(
+    bind=True,
+    max_retries=5,
+    queue=settings.CELERY_ETL_TASK_QUEUE,
+    ignore_result=True,
+)
+def remove_parent_and_child_docs_by_query(
+    self: Task,
+    es_document_name: str,
```

Comment:
Would a more meaningful type on this and elsewhere be `Literal[XYZ, ABC]`?

Reply:
Sure, I've created a new type `ESDocumentNameType` based on `Literal` that contains the valid document type strings. And use in all the places that has `es_document_name` as param